unit-tests/src/org/commcare/android/tests/backend/PrototypeTest.java

```diff
@@ -0,0 +1,55 @@
+/**
+ *
+ */
+package org.commcare.android.tests.backend;
+import org.commcare.android.junit.CommCareTestRunner;
+import org.commcare.android.shadows.SQLiteDatabaseNative;
+import org.commcare.cases.model.Case;
+import org.commcare.util.externalizable.AndroidClassHasher;
+import org.commcare.util.externalizable.AndroidPrototypeFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.annotation.Config;
+
+import static junit.framework.Assert.assertEquals;
+
+
+/**
+ * @author wspride
+ *
+ */
+@Config(shadows={SQLiteDatabaseNative.class}, emulateSdk = 18, application=org.commcare.dalvik.application.CommCareApplication.class)
+@RunWith(CommCareTestRunner.class)
+public class PrototypeTest {
```

Comment:
This isn't compiling. Look at how other robolectric tests have been migrated to robolectric 3. For instance, you shouldn't need some of the `@Config` params because they have been specified in a global config file.